README.md

```diff
@@ -625,6 +625,19 @@ const authMiddleware = createAuthMiddleware({
 })
 ```
 
+**Bearer Token**
+
+This authorizer is responsible for setting the `Authorization` header using the
+`Bearer` scheme. This authorizer expects a `token` field as returned by the buillt-in credentials authenticator.
```

Comment:
Should we add a little section here and above to explain when you'd want to use one vs the other? I just want to make sure we are extremely clear the differences so that others don't get confused on which one to use if they want to use a built-in authorizer.

Reply:
I think the name bearerToken is confusing. How about jwtToken? I can't find any RFC related to it - it just seems to be an accepted standard to use the word Bearer in the header rather than something formal. So perhaps just renaming it and adding a link to https://jwt.io/ for this one and the RFC for the other as well as a bit more info as to the diff?